pkg/db/interface.go

```diff
@@ -46,8 +46,10 @@ type VizierDBInterface interface {
 	DBInit()
 	SelectOne() error
 
-	GetStudyConfig(string) (*api.StudyConfig, error)
+	GetStudyMetrics(string) ([]string, error)
```

Comment:
do GetStudyMetrics really need be exposed in VizierDBInterface? it seems for internal use

Reply:
I think, this is just for Select metrics from studies table.
We use it here: https://github.com/kubeflow/katib/blob/869fdf23b70c602b9aa9b30af5bfaff15a9ae8a5/pkg/db/interface.go#L885

Reply:
if it's only in need in pkg/db/interface.go, you can remove line 49